### PR TITLE
XRDDEV-1395 cache TTL

### PR DESCRIPTION
**Pull request by jansu76**
https://jira.niis.org/browse/XRDDEV-1395

### CODE REVIEW

File: src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/TtlConcurrentMapCacheConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ * ConcurrentMapCache autoconfiguration.
+ *
+ * storeByValue = true is not supported. If that is needed, either change this configuration or use
+ * e.g. Caffeine (perhaps better option)
+ */
+@Configuration
+@Slf4j
+public class TtlConcurrentMapCacheConfiguration {
+
+    public static final int TTL_SECONDS = 60;
```
Comment (Dos4dev) [inline src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/TtlConcurrentMapCacheConfiguration.java:52]: The value used in configuration files can easily be changed, and different profiles can use different values. Mey, I ask, why not get this from the properties file?

Comment (jansu76) [reply src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/TtlConcurrentMapCacheConfiguration.java:52]: Good point. I think I will change this.

Comment (jansu76) [reply src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/TtlConcurrentMapCacheConfiguration.java]: Changed in https://github.com/nordic-institute/X-Road/pull/797/commits/67e80cf72d39b41e7e55ffc56061888f5af28560

Comment (Dos4dev) [reply src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/config/TtlConcurrentMapCacheConfiguration.java]: Thank you for the fast response.

**Review by jhyoty — APPROVED** [review]

**Pull request merged**